apps/omg/lib/omg/state.ex

```diff
@@ -109,7 +101,15 @@ defmodule OMG.State do
                last_deposit_query_result,
                child_block_interval
              ) do
-        _ = Logger.info("Started State, height: #{height_query_result}, deposit height: #{last_deposit_query_result}")
+        _ =
+          Logger.info(
```

Comment:
maybe this should be a metric? is it a relevant log entry?

Reply:
I think this makes sense as a metric :+1:

Reply:
ok with adding a metric, but please keep the log too, I use it quite often and I'd not like to rely on a metric only.